ConfirmInput/README.md

```diff
@@ -0,0 +1,44 @@
+
+# ConfirmInput Interface
+
+**Status**: Proposed
+
+| Name         | ID                                                                |
+| :----------- | :---------------------------------------------------------------- |
+| ConfirmInput | 16653eaf34c921467120f2685d425ff963db5cbb5aa676a62a2e33bfc3f6828a  |
+
+
+## Description
+
+Interface for confirm input.
+
+## Functions
+
+`select` - prints prompt message to the user and returns true or false choice
+
+arguments:
+
+	handlerId: uint32
+    prompt: bytes
```

Comment:
```suggestion
	answerId: uint32 - id of function callback.
	prompt: bytes - utf-8 string to print to the user before input.
```

Reply:
[confirm_input.tar.gz](https://github.com/tonlabs/DeBot-IS-consortium/files/5981436/confirm_input.tar.gz)

ConfirmInput.sol
ConfirmInput.abi.json
ConfirmInput.hpp
Example.sol